### ISSUE TITLE
sanity.system DaaLoadTest_daa3_0 ArrayIndexOutOfBoundsException

### ISSUE THREAD

**Issue by JasonFengJ9** [eclipse/openj9#6146]
Failure link
------------

https://ci.eclipse.org/openj9/job/Test_openjdk12_j9_sanity.system_x86-64_windows_Nightly/62/tapResults/

Optional info
-------------

Failure output
--------------

```
DaaLoadTest_daa3_0
DLT testFailure: testAddPackedDecimal(net.openj9.test.arithmetics.TestArithmeticOperations): null
DLT java.lang.ArrayIndexOutOfBoundsException
DLT 	at java.base/java.math.BigInteger.add(BigInteger.java:1409)
DLT 	at java.base/java.math.BigInteger.add(BigInteger.java:1312)
DLT 	at net.openj9.test.arithmetics.TestArithmeticOperations.testAddPackedDecimal(TestArithmeticOperations.java:10716)
DLT 	at jdk.internal.reflect.GeneratedMethodAccessor4.invoke(Unknown Source)
DLT 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
DLT 	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
DLT 	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
DLT 	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
DLT 	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
DLT 	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
DLT 	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
DLT 	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
DLT 	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
DLT 	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
DLT 	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
DLT 	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
DLT 	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
DLT 	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
DLT 	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
DLT 	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
DLT 	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
DLT 	at org.junit.runners.Suite.runChild(Suite.java:128)
DLT 	at org.junit.runners.Suite.runChild(Suite.java:27)
DLT 	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
DLT 	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
DLT 	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
DLT 	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
DLT 	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
DLT 	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
DLT 	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
DLT 	at org.junit.runner.JUnitCore.run(JUnitCore.java:115)
DLT 	at net.adoptopenjdk.loadTest.adaptors.JUnitAdaptor.executeTest(JUnitAdaptor.java:130)
DLT 	at net.adoptopenjdk.loadTest.LoadTestRunner$2.run(LoadTestRunner.java:182)
DLT 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
DLT 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
DLT 	at java.base/java.lang.Thread.run(Thread.java:831)
DLT testFinished: testAddPackedDecimal(net.openj9.test.arithmetics.TestArithmeticOperations)
```

**Comment by JasonFengJ9** [comment]
https://ci.eclipse.org/openj9/job/Test_openjdk8_j9_extended.system_ppc64le_linux_Nightly/66/tapResults/
https://ci.eclipse.org/openj9/job/Test_openjdk8_j9_extended.system_x86-64_mac_Nightly/65/tapResults/

**Comment by pshipton** [comment]
First failure:
OpenJ9: b9829d7
OMR: 9f5d885
OpenJDK8: 0843c3e - openj9-openjdk-jdk8
OpenJDK11: ad988f0 - openj9-openjdk-jdk11
OpenJDK12: 35a9302 - openj9-openjdk-jdk12

Last passing:
OpenJ9: 3d9ad73
OMR: 6eea707
OpenJDK8: 3f30074 - openj9-openjdk-jdk8
OpenJDK11: 2a60f1f - openj9-openjdk-jdk11
OpenJDK12: e941d38 - openj9-openjdk-jdk12

https://github.com/eclipse/openj9/compare/3d9ad73...b9829d7
https://github.com/eclipse/omr/compare/6eea707...9f5d885
https://github.com/ibmruntimes/openj9-openjdk-jdk8/compare/3f30074...0843c3e
https://github.com/ibmruntimes/openj9-openjdk-jdk11/compare/2a60f1f...ad988f0

**Comment by pshipton** [comment]
@andrewcraik fyi this blocker

**Comment by andrewcraik** [comment]
@pshipton The change was merged in OMR on Friday but looks like the acceptance did not complete in time for these builds. Am waiting on the CI interface to load to check on current status.

**Comment by pshipton** [comment]
@andrewcraik eclipse/omr#4017 didn't fix this problem or #6145. It was merged and promoted on Friday Jun 14, but these problems continue to occur.

**Comment by pshipton** [comment]
Seems to be resolved now.
